app/src/main/java/re/neutrino/kanji_assist/AssistStructureVisualizer.java

```diff
@@ -145,9 +140,25 @@ private TextView recreateTextView(AnyAssistStructure.ViewNode node,
         return result;
     }
 
+    private static CharSequence nodeGetTextToDisplay(AnyAssistStructure.ViewNode node) {
+        final CharSequence text = node.getText();
+        if (isNotEmpty(text))
```

Comment:
maybe use `TextUtils.isEmpty(text)` it will return true if `text` is `null` or empty.